packages/fether-react/src/Send/TxForm/TxDetails/TxDetails.js

```diff
@@ -53,7 +57,7 @@ ${this.renderTotalAmount()}`;
       <div>
         <div className='form_field'>
           <div hidden={!showDetails}>
-            <label htmlFor='txDetails'>Transaction Details (Estimate):</label>
+            <label htmlFor='txDetails'>Transaction Details (estimation):</label>
```

Comment:
nitpicking: if we use this wording, the gas limit is actually not an estimation, it's a hard limit we're putting on the tx.

Reply:
Right, let's remove that estimation :)